sigs/cncf-sigs.md

```diff
@@ -114,6 +114,14 @@ As a starting point let’s be inspired by CNCF OSS Projects and by K8s SIGs.  T
 * Leads projects in the SIG’s area.
 * Has the time and ability to perform deep technical dives on projects.   Projects may include formal CNCF projects or other projects in the area covered by the SIG.
 
+
+#### TOC Liaison
+
+* Streamlines communications between TOC and SIG chairs.
```

Comment:
As part of this, maybe we should add something about helping the SIG set priorities

Reply:
@lizrice will submit suggested changes in the subsequent pr